src/wincng.c

```diff
@@ -1226,14 +1256,25 @@ _libssh2_wincng_rsa_sha1_sign(LIBSSH2_SESSION *session,
     unsigned long cbData, datalen, siglen;
     int ret;
 
+    if(hash_len == SHA_DIGEST_LENGTH)
+      paddingInfo.pszAlgId = BCRYPT_SHA1_ALGORITHM;
```

Comment:
Mixed indentation levels between the if-statements. Should be 4 instead of 2 spaces here.

Reply:
Fixed it in the latest commit.